src/main/kotlin/com/exactpro/th2/check1/rule/RuleFactory.kt

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright 2021-2021 Exactpro (Exactpro Systems Limited)
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.check1.rule
+
+import com.exactpro.th2.check1.SessionKey
+import com.exactpro.th2.check1.StreamContainer
+import com.exactpro.th2.check1.exception.RuleCreationException
+import com.exactpro.th2.check1.exception.RuleInternalException
+import com.exactpro.th2.check1.grpc.CheckRuleRequest
+import com.exactpro.th2.check1.grpc.CheckSequenceRuleRequest
+import com.exactpro.th2.check1.rule.check.CheckRuleTask
+import com.exactpro.th2.check1.rule.sequence.SequenceCheckRuleTask
+import com.exactpro.th2.common.event.Event
+import com.exactpro.th2.common.event.bean.builder.MessageBuilder
+import com.exactpro.th2.common.grpc.ComparisonSettings
+import com.exactpro.th2.common.grpc.Direction
+import com.exactpro.th2.common.grpc.EventBatch
+import com.exactpro.th2.common.grpc.EventID
+import com.exactpro.th2.common.grpc.MessageFilter
+import com.exactpro.th2.common.grpc.RootComparisonSettings
+import com.exactpro.th2.common.grpc.RootMessageFilter
+import com.exactpro.th2.common.message.toJson
+import com.exactpro.th2.common.schema.message.MessageRouter
+import com.google.protobuf.GeneratedMessageV3
+import io.reactivex.Observable
+import mu.KotlinLogging
+import org.slf4j.Logger
+import java.time.Instant
+import java.util.concurrent.ForkJoinPool
+
+class RuleFactory(
+        private val maxEventBatchContentSize: Int,
+        private val streamObservable: Observable<StreamContainer>,
+        private val eventBatchRouter: MessageRouter<EventBatch>
```

Comment:
According to kotin convention, we should use the same size for indentation in the continuation indent and the normal indent. So, it would nice if you reformat the code according that
```suggestion
    private val maxEventBatchContentSize: Int,
    private val streamObservable: Observable<StreamContainer>,
    private val eventBatchRouter: MessageRouter<EventBatch>
```